packages/shared-components/src/components/onto-repository-selector/onto-repository-selector.scss

```diff
@@ -0,0 +1,28 @@
+:root {
+  --onto-repository-selector-color: var(--secondary-color, #373a3c);
+}
+
+:host {
+  display: block;
+}
+
+.onto-repository-selector {
+
+  .onto-dropdown {
+
+    &.closed {
+      .onto-dropdown-button {
+        background-color: #ebebeb;
+        color: var(--onto-repository-selector-color);
+
+        &:hover {
+          background-color: #d4d4d4;
+        }
+      }
+    }
+  }
+
+  .onto-dropdown-menu {
+    max-height: 340px;
```

Comment:
How did you came with this height value? Is it appropriate to move this in the dropdown component?

Reply:
Moved to dropdown component